Dockerfile-ubuntu18.04/README.md

```diff
@@ -1,17 +1,33 @@
-## Build Dockerfile for e2eAIOK workloads
+# Build Dockerfile for Intel® End-to-End AI Optimization Kit
 
+## How to build it
+
+### By default, you can build e2eAIOK docker image with latest nightly-build e2eAIOK PyPI distributions.
 ```
 $ cd Dockerfile-ubuntu18.04
 $ docker build -t e2eaiok-tensorflow . -f DockerfileTensorflow
 $ docker build -t e2eaiok-pytorch . -f DockerfilePytorch
 $ docker build -t e2eaiok-pytorch112 . -f DockerfilePytorch112
 ```
 
-Notice:
-If you need a proxy to build docker, please use below build scripts instead.
+### If you need http and https proxy to build docker image:
 ```
 $ cd Dockerfile-ubuntu18.04
 $ docker build -t e2eaiok-tensorflow . -f DockerfileTensorflow --build-arg http_proxy=http://proxy-ip:proxy-port --build-arg https_proxy=http://proxy-ip:proxy-port
 $ docker build -t e2eaiok-pytorch . -f DockerfilePytorch --build-arg http_proxy=http://proxy-ip:proxy-port --build-arg https_proxy=http://proxy-ip:proxy-port
 $ docker build -t e2eaiok-pytorch112 . -f DockerfilePytorch112 --build-arg http_proxy=http://proxy-ip:proxy-port --build-arg https_proxy=http://proxy-ip:proxy-port
 ```
+
+## How to use the image.
+
+### To start a cluster via one-click script
+``` bash
+git clone https://github.com/intel/e2eAIOK.git
+git submodule update --init --recursive
+cd e2eAIOK; python scripts/start_e2eaiok_docker.py --backend [tensorflow, pytorch, pytorch112] --dataset_path ../ --workers host1, host2, host3, host4 --proxy "http://addr:ip"
+```
+
+### To start a notebook directly
+```
+docker run -it --privileged --network host --device=/dev/dri e2eaiok-pytorch112
+```
```

Comment:
can you also embed this into start_e2eaiok_docker.py? I found by instructing user with cmdline, lots of arguments are missing
see below example, we need to set shm-size, set dataset_path, set workplace set docker_name

["docker", "run",  "--shm-size=100g", "--privileged",  "--network",  "host", "--device=/dev/dri", "-d", "-v", f"{dataset_path}/:/home/vmagent/app/dataset", "-v", f"{current_folder}/:/home/vmagent/app/e2eaiok",  "-w",  "/home/vmagent/app/", "--name", docker_nickname,  docker_name, "/bin/bash", "-c", "\"service ssh start & sleep infinity\""]

Reply:
I think the cmdline can be something like `python scripts/start_e2eaiok_docker.py --backend [tensorflow, pytorch, pytorch112] --jupyter_mode --dataset_path ../ --workers host1, host2, host3, host4 --proxy "http://addr:ip"`

Reply:
Refined.
Can be triggered through this option `jupyter_mode`

python scripts/start_e2eaiok_docker.py --backend [tensorflow, pytorch, pytorch112] --jupyter_mode --dataset_path ../ --workers host1, host2, host3, host4 --proxy "http://addr:ip"